cmd/daemon.go

```diff
@@ -129,6 +130,13 @@ func startDiagnosticsDaemon() {
 	if defaultConfig.FlagForceTLS {
 		options = append(options, nodeutil.OptionMesosStateURL(defaultStateURL.String()))
 	}
+	if runtime.GOOS == "windows" {
+		detectIPScriptPath := os.Getenv("SYSTEMDRIVE") + "\\DCOS\\diagnostics\\detect_ip.ps1"
+		options = append(options, nodeutil.OptionDetectIP(detectIPScriptPath))
```

Comment:
same here, can we use const or variable to define a default path on windows machine?

Reply:
good idea.  I will replace it with a const.